release-notes.txt

```diff
@@ -0,0 +1,80 @@
+BISDN Linux
+4.2.0 Release Notes
+Date: 2021-10-06
+
+! New in this release
+In this release, we add the EdgeCore AS7726-32X as our first Trident3 based 100G switch to the list of supported platforms.
+
+! Bug fixes
+For systemd-networkd, the redundant `/etc/systemd/network/99-enp.network` management interface configuration file was dropped. 
```

Comment:
i think we want to mention here why it was redundant

Reply:
we also might want to either state here or somewhere in our docs how to handle this during an upgrade from 4.1.0, since we do not actually drop anything, we just do not add it anymore